wolfcrypt/src/ed25519.c

```diff
@@ -388,18 +490,81 @@ int wc_ed25519ph_sign_msg(const byte* in, word32 inLen, byte* out,
 
 #ifdef HAVE_ED25519_VERIFY
 
+/*
+   sig        is array of bytes containing the signature
+   sigLen     is the length of sig byte array
+   key        Ed25519 public key
+   return     0 on success
+   type       variant to use -- Ed25519, Ed25519ctx, or Ed25519ph
+   context    extra signing data
+   contextLen length of extra signing data
+*/
+static int ed25519_verify_msg_init_with_sha(const byte* sig, word32 sigLen, ed25519_key* key, wc_Sha512 *sha,
```

Comment:
Nit pick... Can we fix the line length here.

Reply:
done